lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2252,12 +2332,13 @@ constructTransaction ctx genChange knownPools getPoolStatus (ApiT wid) body = do
             pure (sel, sel', estMin)
 
         tx <- liftHandler
-            $ W.constructTransaction @_ @s @k @n wrk wid txCtx sel
+            $ W.constructTransaction @_ @s @k @n wrk wid txCtx' sel
 
         pure $ ApiConstructTransaction
             { transaction = ApiT tx
             , coinSelection = mkApiCoinSelection
                 (maybeToList deposit) (maybeToList refund) Nothing md sel'
+            , mintedBurned = Nothing
```

Comment:
Should this always be `Nothing`?

Reply:
oh, you are right. I forgot to update this. Of course, when minting/burning occurs it should not be Nothing. Added impl and expectations in mint/burn integration testing showing the case. Thanks for this catch.